lib/bundler/audit/cli.rb

```diff
@@ -60,10 +61,10 @@ def check
       end
 
       desc 'update', 'Updates the ruby-advisory-db'
-      def update
+      def update(quiet = false)
```

Comment:
Yes although I would err on the side of YAGNI. It's the only option `update` accepts right now. If you really want it, I will change it.